lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -191,14 +191,20 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
 
 private:
   static StringRef getFilenameFromDC(const DeclContext *DC) {
-    if (auto LF = dyn_cast<LoadedFile>(DC))
+    if (auto *LF = dyn_cast<LoadedFile>(DC))
       return LF->getFilename();
-    if (auto SF = dyn_cast<SourceFile>(DC))
+    if (auto *SF = dyn_cast<SourceFile>(DC))
       return SF->getFilename();
-    else if (auto M = dyn_cast<ModuleDecl>(DC))
+    if (auto *M = dyn_cast<ModuleDecl>(DC)) {
+      auto Files = M->getFiles();
+      if (Files.size() == 1 && Files[0]) {
+        StringRef PI = Files[0]->getParseableInterface();
```

Comment:
Sorry, that's not what I meant. I meant change ModuleDecl::getModuleFilename to refer to the swiftinterface, and if I'd thought it through before I would have said that in the first place!